### ISSUE TITLE
Experiencing `ValueError`: Characteristic notifications already started

### ISSUE THREAD

**Issue by dhruvkb** [rhyst/linak-controller#77]
`linak-controller` is raising the following error.

```
Connected 84F900A3-AB7F-C330-501D-575AB6D66797
Capabilities: {'memSize': 3, 'autoUp': False, 'autoDown': False, 'bleAllow': True, 'hasDisplay': False, 'hasLight': False}

Something unexpected went wrong:
Traceback (most recent call last):
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/linak_controller/main.py", line 204, in main
    client = await connect()
             ^^^^^^^^^^^^^^^
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/linak_controller/main.py", line 46, in connect
    await Desk.initialise(client)
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/linak_controller/desk.py", line 30, in initialise
    user_id = await DPGService.dpg_command(client, DPGService.DPG.CMD_USER_ID)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/linak_controller/gatt.py", line 200, in dpg_command
    await cls.DPG.subscribe(client, callback)
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/linak_controller/gatt.py", line 27, in subscribe
    await client.start_notify(cls.uuid, callback)
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/bleak/__init__.py", line 834, in start_notify
    await self._backend.start_notify(characteristic, wrapped_callback, **kwargs)
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/bleak/backends/corebluetooth/client.py", line 357, in start_notify
    await self._delegate.start_notifications(characteristic.obj, callback)
  File "/opt/pipx/venvs/linak-controller/lib/python3.11/site-packages/bleak/backends/corebluetooth/PeripheralDelegate.py", line 217, in start_notifications
    raise ValueError("Characteristic notifications already started")
ValueError: Characteristic notifications already started
```

<img width="2672" alt="Screenshot 2023-11-05 at 1 24 02 PM" src="https://github.com/rhyst/linak-controller/assets/16580576/211d88ef-ed6e-469d-b3e4-ab827cd0a7f6">

## Specs

- **Package name:** `linak-controller`
- **Package version:** 1.1.0
- **Python version:** 3.11.6
- **Installed via:** pipx
- **OS name:** macOS
- **OS version:** 14.1 (23B74)
- **Desk name:** Ikea Idåsen

**Comment by snok3r-krk** [comment]
I have similar issue
![image](https://github.com/rhyst/linak-controller/assets/22940290/eafe18cf-008d-47a2-9b36-f215bc9b6b98)

Specs
Package name: linak-controller
Package version: 1.1.0
Python version: 3.11
Installed via: pip3
OS name: macOS
OS version: 14.1.1
Desk name: Ikea Idåsen

**Comment by snok3r-krk** [comment]
It works with `1.1.1-dev` 🎈

**Comment by rhyst** [comment]
Released 1.1.1
